servers/src/main/java/tachyon/master/file/meta/InodeTree.java

```diff
@@ -270,8 +270,8 @@ public CreatePathResult createPath(TachyonURI path, CreatePathOptions options)
 
     // Create the final path component. First we need to make sure that there isn't already a file
     // here with that name. If there is an existing file that is a directory and we're creating a
-    // directory, update persistence property of the directories if needed, otherwise, nothing needs
-    // to be done.
+    // directory, update persistence property of the directories if needed, otherwise, throw
```

Comment:
when persistence property needs to be updated, shall we fail the mkdir operation? 
Currently in my impl we will only fail mkdir when 1) file already exists 2) persistence info is up to date